src/main/java/nl/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,77 @@
+package nl.novi.TechItEasy.controllers;
+
+import nl.novi.TechItEasy.exceptions.InvalidTitle;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.ArrayList;
+
+@RestController
+public class TelevisionController {
```

Comment:
Ik zou hier een Requetsmapping plaatsen, dat scheelt veel typwerk en foutjes, zo gebruik je /televisions en /television hieronder en is dat niet heel gebruiksvriendelijk.